### PR TITLE
[Test] add a test for hydrating nested static vars

### PR DESCRIPTION
**Pull request by llllllluc**

### CODE REVIEW

File: contracts/warp-resolver/src/tests.rs [hunk 1]
```diff
@@ -355,77 +355,92 @@ fn test_hydrate_vars_nested() {
...
-
-    match hydrated_vars[1].clone() {
+    let hydrated_var3 = hydrated_vars[1].clone();
+    match hydrated_var3.clone() {
         Variable::Static(static_var) => {
-            // let decoded_val = base64::decode(static_var.value).unwrap();
-            println!("Decoded Val: {}\n\n\n", static_var.value);
-            // Decoded Val: {"swap":{"offer_asset":{"info":{"native_token":{"denom":"example_denom"}},"amount":"$warp.variable.var1"},"max_spread":"0.01","to":"your_address_here"}}
-            // as you can see, var1 is replaced to static_value_1 as expected
+            assert_eq!(
```
Comment (llllllluc) [inline contracts/warp-resolver/src/tests.rs:407]: whether var.encode = true or false doesn't affect here, it only affect var's value when injecting to msg during msg hydration

**Comment by vladjdk** [comment]
lgtm, merging, thanks!

**Comment by llllllluc** [comment]
close this one and include the change in https://github.com/terra-money/warp-contracts/pull/59

**Comment by llllllluc** [comment]
close this one and include the change in https://github.com/terra-money/warp-contracts/pull/59

**Pull request closed without merging**